apps/codeforafrica/src/lib/api.ghost.js

```diff
@@ -0,0 +1,101 @@
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+function ghostAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5",
```

Comment:
```suggestion
    version: process.env.GHOST_API_VERSION || "v5.0",
```
I think Ghost Content API [version](https://ghost.org/docs/faq/api-versioning/) is always in `v{major}.{minor}` format.

Reply:
According to [this](https://ghost.org/docs/content-api/javascript/#:~:text=version%20%2D%20should%20be%20set%20to%20%E2%80%98v3%E2%80%99) , we should set it to v3. Perhaps v3.0?

Reply:
In the **Working Example** section in the same link you shared, they're using `v5.0`. Not sure why they're saying it should be `v3` here.

Either way, as long as it works and it's in `v{major}.{minor}`, I'm happy.